packages/llvm5_polly.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Llvm5_polly < Package
+  description 'Polly is a high-level loop and data-locality optimizer and optimization infrastructure for LLVM.'
+  homepage 'http://polly.llvm.org/'
+  version '5.0.1'
+  source_url 'http://releases.llvm.org/5.0.1/polly-5.0.1.src.tar.xz'
+  source_sha256 '9dd52b17c07054aa8998fc6667d41ae921430ef63fa20ae130037136fdacf36e'
+
+  depends_on 'buildessential'
+  depends_on 'cmake'
+  depends_on 'llvm5'
+
+  def self.build
+    system "mkdir -p mybuilddir"
+    Dir.chdir "mybuilddir" do
+ 	    system "cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release .."
+        system "cmake --build . --  -j#{CREW_NPROC}"
+    end
+  end
+
+  def self.install
+    Dir.chdir "mybuilddir" do
+       system "cmake -DCMAKE_INSTALL_PREFIX=#{CREW_DEST_DIR}/usr/local -P cmake_install.cmake"
```

Comment:
Missing `.` or `..` target directory?

Reply:
In these cases, the install command works fine - there's no need to specify the directory if the -P option is specified I think (which gives the path to the install script.

Reply:
It's equivalent to running `make install` in the build directory. I'm not sure why the original creator of the llvm package did it that way, but I just followed his/her lead.

Reply:
OK great.  Just wanted to make sure.  Nice to know.